lms/models/application_instance.py

```diff
@@ -61,58 +64,58 @@ class ApplicationInstance(BASE):
         nullable=False,
     )
 
-    #: A unique identifier for the LMS instance.
+    # A unique identifier for the LMS instance
     tool_consumer_instance_guid = sa.Column(sa.UnicodeText, nullable=True)
 
-    #: The LMS product name, e.g. "canvas" or "moodle".
+    # The LMS product name, e.g. "canvas" or "moodle"
     tool_consumer_info_product_family_code = sa.Column(sa.UnicodeText, nullable=True)
 
-    #: A plain text description of the LMS instance, e.g. "University of Hypothesis"
+    # A plain text description of the LMS instance, e.g. "Uni of Hypothesis"
     tool_consumer_instance_description = sa.Column(sa.UnicodeText, nullable=True)
 
-    #: The URL of the LMS instance, e.g. "https://hypothesis.instructure.com".
+    # The URL of the LMS instance, e.g. "https://hypothesis.instructure.com"
     tool_consumer_instance_url = sa.Column(sa.UnicodeText, nullable=True)
 
-    #: The name of the LMS instance, e.g. "HypothesisU".
+    # The name of the LMS instance, e.g. "HypothesisU"
     tool_consumer_instance_name = sa.Column(sa.UnicodeText, nullable=True)
 
-    #: An contact email, e.g. "System.Admin@school.edu"
+    # An contact email, e.g. "System.Admin@school.edu"
     tool_consumer_instance_contact_email = sa.Column(sa.UnicodeText, nullable=True)
 
-    #: Version of the LMS, e.g. "9.1.7081"
+    # Version of the LMS, e.g. "9.1.7081"
     tool_consumer_info_version = sa.Column(sa.UnicodeText, nullable=True)
 
-    #: This Canvas custom variable substitution $Canvas.api.domain.
-    #: We request this in our config.xml file and name it "custom_canvas_api_domain":
-    #:
-    #: https://github.com/hypothesis/lms/blob/5394cf2bfb92cb219e177f3c0a7991add024f242/lms/templates/config.xml.jinja2#L20
-    #:
-    #: See https://canvas.instructure.com/doc/api/file.tools_variable_substitutions.html
+    # This Canvas custom variable substitution $Canvas.api.domain. We request
+    # this in our config.xml file and name it "custom_canvas_api_domain":
+    #
+    # https://github.com/hypothesis/lms/blob/5394cf2b/lms/templates/config.xml.jinja2#L20
```

Comment:
The comment was guarding against a collision

Reply:
I'm not entirely sure if you are joking or not, but with 8 chars we should have a collision chance of 1 in 4 billion or so, so we're probably fine?

I can put it back?